tests/test_validators.py

```diff
@@ -31,6 +32,50 @@ def test_invalid_url(self):
             self.form.process(url=url)
             self.assertEqual(self.form.validate(), False)
 
+    def test_public_email_domain(self):
+        with self.app.test_request_context('/'):
+            # both valid
+            self.webmail_form.process(
+                webmail_domain='gmail.com',
+                not_webmail_domain='foobar.com'
+            )
+            self.assertTrue(self.webmail_form.validate())
+
+            # Emoji domain, should not raise any exception
+            self.webmail_form.process(
+                webmail_domain='gmail.com',
+                not_webmail_domain='i❤.ws'
```

Comment:
How is this working if not declared as a Unicode string? You don’t have Unicode literals enabled.